apps/client/src/views/cuesheet/cuesheet-table-elements/EditableCell.tsx

```diff
@@ -1,27 +1,36 @@
 import { ChangeEvent, memo, useCallback, useEffect, useRef, useState } from 'react';
+import { Input } from '@chakra-ui/react';
 import { getHotkeyHandler } from '@mantine/hooks';
 
 import { AutoTextArea } from '../../../common/components/input/auto-text-area/AutoTextArea';
 
 interface EditableCellProps {
   value: string;
   handleUpdate: (newValue: string) => void;
+  isMultiLine: boolean;
 }
 
 const EditableCell = (props: EditableCellProps) => {
-  const { value: initialValue, handleUpdate } = props;
+  const { value: initialValue, handleUpdate, isMultiLine } = props;
```

Comment:
question: did you look into using the `useReactiveInput` hook? it could help with the submit logic